meta-mender-core/recipes-mender/mender-client/mender-client-cpp.inc

```diff
@@ -0,0 +1,17 @@
+require mender-client.inc
+
+RPROVIDES:${PN} = "mender-client"
```

Comment:
You probably need to specify this:

```
RPROVIDES:mender-update = "mender-client"
RDEPENDS:mender-update = "mender-auth"
```

since it is that package which provides the complete thing (it depends on mender-auth, so will pull in that too).